src/NLog/Layouts/JsonLayout.cs

```diff
@@ -108,12 +108,12 @@ protected override void InitializeLayout()
             base.InitializeLayout();
             if (IncludeMdc)
             {
-                ThreadAgnostic = false;
+                base.ThreadAgnostic = false;
             }
-#if !SILVERLIGHT && !NETSTANDARD || NETSTANDARD1_3
```

Comment:
good one, will make a new PR for it. Thanks.

right now I will that the same `#if` as the `IncludeMdlc` check